cmake/Platform/Linux/Packaging/snapcraft.yaml.in

```diff
@@ -0,0 +1,64 @@
+name: o3de
+version: '${O3DE_INSTALL_VERSION_STRING}'
+summary: O3DE Engine
+description: |
+  O3DE Engine
+confinement: classic
+base: core22
+
+parts:
+  test-o3de:
+    plugin: dump
+    source: ./${CPACK_PACKAGE_NAME}
+    source-type: local
+    stage-packages:
```

Comment:
Just checking, but do these libraries need to match the dependencies of what we're using to build? I note that `libunwind-dev` among others are not in this list, as noted in the `Packaging_linux.cmake` file for the DEB package

Reply:
Any mismatches from that list are either it was only used for building or it was pulled in from using core22. I'll double check this specific library though in case it's something I just overlooked.